woocommerce/woocommerce-android
# Edit variations: Shipping

This PR adds the shipping editing functionality to Variation details. The current shipping screen functionality has been extracted from ProductDetailViewModel into a separate one so that it could be reused for variations. 

Also some common code has been extracted into a base fragment `BaseProductEditFragment`, which is inherited by all the editor fragments.

![image](https://user-images.githubusercontent.com/1522856/90784510-a28f6380-e301-11ea-9b94-51045c038a81.png)

**To test:**
1. Go to Products -> Select a physical product -> Tap on Shipping
2. Verify that shipping editing works 
3. Go back to Products
4. Open a variable product -> Tap on Shipping
5. Verify that shipping editing works (shipping class is not editable on purpose)
